HoaryFox/KarambaConnect/Element.cs

```diff
@@ -0,0 +1,53 @@
+using System.Collections.Generic;
+using Karamba.CrossSections;
+using Karamba.Elements;
+using Karamba.Geometry;
+using Karamba.Utilities;
+using STBReader;
+using STBReader.Member;
+using STBReader.Model;
+
+namespace HoaryFox.KarambaConnect
+{
+    public class Element
```

Comment:
Codacy found an issue: [Add a 'protected' constructor or the 'static' keyword to the class declaration.](https://app.codacy.com/gh/hrntsm/HoaryFox/pullRequest?prid=6405207)

Reply:
fix